elpis/engines/common/objects/model.py

```diff
@@ -33,10 +33,36 @@ def load(cls, base_path: Path):
     def status(self):
         return self.config['status']
 
+    @property
+    def log(self):
+        return self.config['log']
+
+    @property
+    def stage_status(self):
+        return self.config['stage_status']
+
     @status.setter
     def status(self, value: str):
         self.config['status'] = value
 
+    @log.setter
+    def log(self, value: str):
+        self.config['log'] = value
+
+    @stage_status.setter
+    def stage_status(self, vals: Tuple[str, str, str]):
```

Comment:
In lieu of comments, is there a more descriptive name for the variable than vals.

Reply:
yes, added in 35267e3